bindings/experimental/distrdf/python/DistRDF/HeadNode.py

```diff
@@ -95,19 +94,35 @@ def build_ranges(self):
 class TreeHeadNode(Node.Node):
     """
     The head node of a computation graph where the RDataFrame data source is
-    a TTree. This head node is responsible for the following RDataFrame constructor::
+    a TTree or a TChain. This head node is responsible for the following
+    RDataFrame constructors::
```

Comment:
Double colon

Reply:
Double colon is how Sphinx parses a code block (doxygen would expect a ~~~{.py}) . It's still here and there in DistRDF, when we'll move to fully utilise doxygen will be changed